opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -1006,7 +1012,7 @@ export const RUNTIME_ATTRIBUTES = {
       // eslint-disable-next-line no-use-before-define
       const users = await elPaginate(context, user, READ_INDEX_INTERNAL_OBJECTS, {
```

Comment:
could we load users from cache too here ?

Reply:
Updated